### ISSUE TITLE
[Feature Request] Add Turn Left and Turn Right as a bind option for mouse buttons

### ISSUE THREAD

**Issue by IceSpyLittleEye** [fabiangreffrath/crispy-doom#725]
I noticed we have Move and Strafe. Would it be possible to also add in Turn Left and Turn Right to the additional mouse buttons as well? I saw Gamepad/Joystick and Keyboard both had it as an option and wondered if this could be done for mouse buttons as well.

Thank you for your time.

**Comment by fabiangreffrath** [comment]
Are you really playing with turning bound to mouse keys in any other port? I mean, you could turn much more precisely by moving the mouse or not? I am just asking to check if this is actually useful or merely requested for completeness.

**Comment by IceSpyLittleEye** [comment]
> 
> 
> Are you really playing with turning bound to mouse keys in any other port? I mean, you could turn much more precisely by moving the mouse or not? I am just asking to check if this is actually useful or merely requested for completeness.

This would be more of a personal useful thing for me alone probably. I use my side buttons - Mouse 4 for Turn Right and Mouse 5 for Turn Left to help with comfort. An example would be SR50 which requires holding many keys at once (including Turn Left/Right), which I was finding my hand was straining. I tried numerous setups (Strafe On Mouse 2 being a big help) but eventually every position was still causing awkward strain on my keyboard hand.

Decided to rebind my two side buttons (big help in general comfort for me in gaming) with Turn Right/Left on M4/M5. I was no longer straining my hand at all and could play long periods as well as casually "speedrun". I used GZDoom because I mostly played WADS casually, though I still did fake speedruns that were not official, but for personal fun.

I decided I wanted to try proper speedrunning. I picked up Crispy as it was recommended and I saw that I could bind my side buttons too, unfortunately there was no Turn Left/Right option. Everything else is perfect.
